src/gasPriceOracle/adapters/arbitrum.ts

```diff
@@ -0,0 +1,21 @@
+import { BigNumber, providers, utils as ethersUtils } from "ethers";
+import { GasPriceEstimate } from "../types";
+import { eip1559 } from "./ethereum";
+
+// Arbitrum Nitro implements EIP-1559 pricing, but the priority fee is always refunded to the caller. Further,
+// ethers typically hardcodes the priority fee to 1.5 Gwei. So, confirm that the priority fee supplied was 1.5
+// Gwei, and then drop it to 1 Wei. Reference: https://developer.arbitrum.io/faqs/gas-faqs#q-priority
+export async function eip1559_arbitrum(provider: providers.Provider, chainId: number): Promise<GasPriceEstimate> {
```

Comment:
Have you tested that 1 wei proirity fee works?  This is a really strange gas fee structure...

Reply:
Just circling back to this one - here's a sample txn where I used this updated gasPriceOracle in the relayer to fetch the gas price: https://arbiscan.io/tx/0x50fe76875656fc58526861e9f20a78a8f95876f6706ea02c69fd4a26b9c79194

The fees supplied by the gasPriceOracle were 0.1 Gwei and 1 Wei respectively, and the maxFeePerGas in the fill transaction was 0.36 Gwei (after scaling in the relayer-v2 getGasPrice() function). Ultimately, a price of 0.1 Gwei was charged, and the rest was refunded to the relayer. So the on-chain behaviour seems consistent with the description in the docs :+1: